quantus/metrics/faithfulness/faithfulness_correlation.py

```diff
@@ -139,7 +139,7 @@ def __init__(
 
         # Save metric-specific attributes.
         if perturb_func is None:
-            perturb_func = baseline_replacement_by_indices
```

Comment:
Let's discuss: where in the code should we make it explicit for the user that they no longer can use any other perturb function that `batch_baseline_replacement_by_indices`.